src/arrayMethodSort.js

```diff
@@ -4,8 +4,21 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => (String(a) > String(b) ? 1 : -1),
+  ) {
+    for (let i = 0; i < this.length; i++) {
+      for (let j = 1; j < this.length; j++) {
```

Comment:
The inner loop should start from `i + 1` instead of `1` to avoid unnecessary comparisons and improve efficiency. This will ensure that the already sorted portion of the array is not re-evaluated.